nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/test/java/org/nuxeo/ecm/platform/rendition/service/RenditionFeature.java

```diff
@@ -19,35 +19,52 @@
 
 package org.nuxeo.ecm.platform.rendition.service;
 
+import java.io.Serializable;
+import java.util.List;
+
 import org.nuxeo.directory.test.DirectoryFeature;
+import org.nuxeo.ecm.core.api.DocumentModel;
 import org.nuxeo.ecm.core.test.CoreFeature;
 import org.nuxeo.ecm.core.test.DefaultRepositoryInit;
 import org.nuxeo.ecm.core.test.annotations.Granularity;
 import org.nuxeo.ecm.core.test.annotations.RepositoryConfig;
+import org.nuxeo.ecm.platform.test.NuxeoLoginFeature;
+import org.nuxeo.ecm.platform.usermanager.UserManager;
+import org.nuxeo.runtime.api.Framework;
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
 import org.nuxeo.runtime.test.runner.RunnerFeature;
 import org.nuxeo.transientstore.test.TransientStoreFeature;
 
 /**
  * @since 7.3
  */
-@Features({ CoreFeature.class, DirectoryFeature.class, TransientStoreFeature.class })
+@Features({ CoreFeature.class, DirectoryFeature.class, NuxeoLoginFeature.class, TransientStoreFeature.class })
```

Comment:
Is DirectoryFeature required as it is included indirectly by NuxeoLoginFeature?

Reply:
In this case we can totally say no because rendition doesn't use explicitly directories. Will remove it.